pkg/minikube/machine/cache_images.go

```diff
@@ -651,7 +651,7 @@ func RemoveImages(images []string, profile *config.Profile) error {
 			err = removeImages(cruntime, images)
 			if err != nil {
 				failed = append(failed, m)
-				klog.Warningf("Failed to remove images for profile %s %v", pName, err.Error())
+				klog.Errorf("Failed to remove images for profile %s %v", pName, err.Error())
```

Comment:
@yxxhero I think it is better to use out package for this.
how about keeping the klog.Warninngf

but also add a out.Warn("..."

Reply:
ok, I will change the code.